src/test_application/DebugObject.cpp

```diff
@@ -207,6 +212,16 @@ void DebugCameraController::update_physics_post()
     Matrix4 &xform = m_scene.reg_get<ACompTransform>(m_ent).m_transform;
     Matrix4 const& xformTgt = m_scene.reg_get<ACompTransform>(m_orbiting).m_transform;
 
+    using osp::active::SysPhysics_t;
+    // Compute Center of Mass of target, if it's a rigid body
+    auto [rbEnt, pCompRb] = SysPhysics_t::find_rigidbody_ancestor(m_scene, m_orbiting);
+    Vector3 comOset{0.0f};
+    if (pCompRb != nullptr)
+    {
+        comOset = xformTgt.transformVector(pCompRb->m_centerOfMassOffset);
+    }
```

Comment:
```
Vector3 com0set =     pCompRb
                               ? xformTgt.transformVector(pCompRb->m_centerOfMassOffset);
                               : Vector3{0.0f};
```

Reply:
I'm going to leave this one as is since that way basically takes the same number of lines but is just less common and obvious compared to the way it is now. If it easily fit on one line I'd use a ternary conditional but it's kind of long and messy

Reply:
Do consider the benefit of being able to mark the variable as const, and initialize it once and only once, instead of multiple places how you are now.

Reply:
Hmm fair point, I'll consider it